### PR TITLE
Update Part 3c Mongo Cluster Onboarding

### PR DESCRIPTION
**Pull request by gwpantazes**
Update mongo onboarding steps and mongoose code sample

- Update steps and screenshot because things have changed
- Add an important NB about checking whether the path is pointing to test or admin (I lost hours to this.). The default is apparently now admin, so people should switch it to test.
- Remove deprecation warnings from the code
    - Handle promises. Unresolved/uncaught promises were throwing warnings.
    - Use `useUnifiedTopology` to fix a deprecation error discussed at https://github.com/Automattic/mongoose/issues/8156

### CODE REVIEW

**Comment by mluukkai** [comment]
great improvements, however I can not merge quite yet since .finally is not currently covered in text.

File: src/content/3/en/part3c.md [hunk 1]
```diff
@@ -161,10 +167,11 @@ const note = new Note({
...
 })
 
-note.save().then(response => {
-  console.log('note saved!')
-  mongoose.connection.close()
-})
+note.save()
+  .then(response => console.log('note saved!'))
+  .catch(err => console.error(`note save error: ${err.message}`))
+  .finally(() => mongoose.connection.close());
```
Comment (gwpantazes) [inline src/content/3/en/part3c.md]: @mluukkai Could we avoid calling `finally` here by just doing the following?
```
note.save()
  .then(response => console.log('note saved!'))
  .catch(err => console.error(`note save error: ${err.message}`))

mongoose.connection.close());
```

**Comment by gwpantazes** [comment]
@mluukkai I've removed the use of `finally`. Now the mongoose close code is closer to how it looked originally, in the `then` response callback.

**Comment by marklaatikainen** [comment]
Pull request has conflicts, so closing it now. If needed, PR can be reopened after conflicts are resolved.

**Pull request closed without merging**